source/threads.tex

```diff
@@ -4833,7 +4833,7 @@
     while (p && p->t != t)
       p = p->next;
 
-    return shared_ptr<node>(move(p));
+    return shared_ptr<node>(std::move(p));
```

Comment:
Why are we explicitly converting a `shared_ptr<node>` to `shared_ptr<node>` in the first place? What's wrong with `return p;`?

Reply:
`return p;` is a more reasonable approach. (I'd be happy if it could be changed to this)

Reply:
Yeah, let's go ahead and change it. We like to show a variety of styles in examples, but writing something long and complicated when something short and simple will do isn't style, it's obfuscation.